samples/gltf_viewer.cpp

```diff
@@ -893,6 +893,8 @@ int main(int argc, char** argv) {
                                     "d.shadowmap.display_shadow_texture_channel"), 0, 3);
                     ImGui::Unindent();
                 }
+                ImGui::Checkbox("Combine Multiview Images",
```

Comment:
What happens when this is unchecked?  Is it just a black screen?

Maybe the flow could be that if multiview is enabled, then this is always on for gltf_viewer. wdyt?

Reply:
It just displays the left view, not crashing.

The flag is enabled by default for now, so gltf_viewer shows side by side image. However, the default will be set to false once they can take multi-layer swapchain.

Reply:
Also I think it's still good to let users switch this flag because they can realize the final image is combined by the debug flag, not a single image originally.

I think this option should be visible only when the multiview is enabled. Let me update it.